samples/net/sockets/echo_server/src/tcp.c

```diff
@@ -339,6 +347,26 @@ static void process_tcp6(void)
 	quit();
 }
 
+static void print_stats(struct k_work *work)
+{
+	struct data *data = CONTAINER_OF(work, struct data, tcp.stats_print);
+	int total_received = atomic_get(&data->tcp.bytes_received);
+
+	if (total_received) {
+		if (total_received < 1024) {
```

Comment:
Hmm this might work if the `STATS_TIMER` was set to 1 second, but currently any transfer between `1024` and `1024 * STATS_TIMER` will be reported as 0 KiB/sec. 
Perhaps the byterate should be calculated first, and then depending on the result it shall be decided whether to print bytes or kilobytes per second? Or  just make the if `(total_received < 1024 * STATS_TIMER)`, shorter but IMO less readable.

Reply:
As @rlubos hints, the test apparently should be `if (total_received / STATS_TIMER < 1024)`.